### PR TITLE
init: Add Samsung modifications from /init.rc

### PR DESCRIPTION
**Pull request by derfelot**

### CODE REVIEW

File: config/init/Android.mk [hunk 1]
```diff
@@ -43,6 +43,15 @@ LOCAL_MODULE_CLASS := ETC
 LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR_ETC)/init
 include $(BUILD_PREBUILT)
 
+include $(CLEAR_VARS)
+LOCAL_MODULE := init.exynos9820.root
+LOCAL_SRC_FILES := init.exynos9820.root.rc
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_STEM := init.exynos9820.root.rc
+LOCAL_MODULE_CLASS := ETC
+LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR_ETC)/init
```
Comment (cryptomilk) [inline config/init/Android.mk]: I would do it like we have it with yoshino, install it to `$(TARGET_OUT_VENDOR_ETC)/init/hw`.

File: config/init/init.exynos9820.rc [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import /vendor/etc/init/init.exynos9820.root.rc
```
Comment (cryptomilk) [inline config/init/init.exynos9820.rc]: `import /vendor/etc/init/hw/init.exynos9820.root.rc`

**Review by cryptomilk — CHANGES_REQUESTED** [review]

File: config/init/init.exynos9820.rc (hunk 2 above)
Comment (derfelot) [reply config/init/init.exynos9820.rc]: done

File: config/init/Android.mk (hunk 1 above)
Comment (derfelot) [reply config/init/Android.mk]: done

**Review by cryptomilk — APPROVED** [review]

**Pull request merged**